theano/sandbox/cuda/__init__.py

```diff
@@ -19,6 +19,15 @@
 
 from theano.tensor.basic import register_transfer
 
+# Check if theano is being imported from a child process
```

Comment:
We need to be able to import the cuda and gpuarray module while not using them for computation.

So you must put them in the use() (for cuda module) and in init_dev() (for the gpuarray module) function. Not at the top level of those module.